libr/core/cmd_type.c

```diff
@@ -1476,139 +1466,15 @@ static int cmd_type(void *data, const char *input) {
 			break;
 		}
 	} break;
-	// ta - link immediate type offset to an address
```

Comment:
Imho aht makes more sense than ahO .. or at least aho.

Also we can keep ta for 2 releases with a deprectation warning. Otherwise users will complain instead of giving feedback

Reply:
This is a good choice, in my opinion

Reply:
aht and aho are taken, though

Reply:
ahm (for "mark") and ahl (for "link") might be a good candidate, . @radare